sources/bloodhound/src/bin/bottlerocket-checks/checks.rs

```diff
@@ -22,15 +22,22 @@ pub struct BR01010101Checker {}
 
 impl Checker for BR01010101Checker {
     fn execute(&self) -> CheckerResult {
-        let module_result = check_file_contains!(
-            PROC_MODULES_FILE,
-            &[" udf,"],
-            "unable to parse modules to check for udf",
-            "udf is currently loaded"
-        );
-        if module_result.status != CheckStatus::PASS {
+        let mut module_result = CheckerResult::default();
+
+        // Make sure UDF isn't already loaded
+        if let Ok(found) = look_for_string_in_file(PROC_MODULES_FILE, " udf,") {
```

Comment:
It would be slightly better to check for `"udf "` - `" udf,"` is matching a line for a module that's a dependency of udf, and only matches if udf is the first module in that list.

However, it seems worth adding a different function, like `look_for_word_in_file` that calls `split_ascii_whitespace` on the input and checks for an exact token match instead of using `contains`, to avoid false positives for short inputs like these.